### ISSUE TITLE
Optimize AUC metric on compile function with R Keras API

### ISSUE THREAD

**Issue by dsaada** [fchollet/deep-learning-models#105]
Hi,

I would like to optimize the metric **AUC with R keras API**, it's possible?

the best way will be:

> model %>% compile(
  loss = 'categorical_crossentropy',
  optimizer = optimizer_rmsprop(),
  metrics = c('auc')
)

but of course, it does not work.

Thanks,
David
